### PR TITLE
Nfdiv 1322

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1322

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/functional/features/sole-conditional-order.feature [hunk 1]
```diff
@@ -49,3 +49,38 @@ Feature: Sole conditional order
...
+    Then the page should include "No files uploaded"
+    When I upload the file "fixtures/larry-the-cat.jpg"
+    Then I wait until the page contains image "larry-the-cat.jpg"
+    And I click "Delete"
+    And I wait until the page doesn't contain "larry-the-cat.jpg"
+    And I select "I cannot upload some or all of my documents"
+    Then the page should include "You can post your documents to the court"
+
+    When I click "Continue"
+    Then the page URL should be "/hub-page"
```
Comment (lara-mpt) [inline src/test/functional/features/sole-conditional-order.feature:86]: Can this just be one test? - considering our tests were failing before because we had so many

Comment (velimd) [reply src/test/functional/features/sole-conditional-order.feature:86]: sure, my idea was i was going to add flaky to this as one of them is an upload test.

Comment (lara-mpt) [reply src/test/functional/features/sole-conditional-order.feature:86]: ah yeah, happy if you want to do that then

File: src/main/steps/applicant1/provide-information-to-the-court/template.njk [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            <strong>{{ maximumFileSize }}</strong> {{ fileSize }}
+          </p>
+
+          <div class="govuk-form-group {{ 'hidden' if not isClarificationAmendableState }}">
+            <div class="govuk-panel upload hidden">
+              <span class="upload-arrow-icon govuk-!-margin-bottom-5"></span>
+              <div class="govuk-form-group">
+                <p class="govuk-body govuk-!-margin-bottom-0">
+                  <span id="upload"></span>
+                  <strong>{{ orStr }}</strong><br>
```
Comment (lara-mpt) [inline src/main/steps/applicant1/provide-information-to-the-court/template.njk:117]: what's orStr, can't see it anywhere else?

Comment (velimd) [reply src/main/steps/applicant1/provide-information-to-the-court/template.njk:117]: just the string 'or', its in the upload content file

File: src/test/functional/features/sole-conditional-order.feature (hunk 1 above)
Comment (velimd) [reply src/test/functional/features/sole-conditional-order.feature:86]: will do just want to see if it passes before i add the tag

**Review by lara-mpt — APPROVED** [review]

**Review by danielbe11 — APPROVED** [review]

**Review by danielbe11 — APPROVED** [review]

File: src/main/app/case/from-api-format.test.ts [hunk 3]
```diff
@@ -314,4 +314,15 @@ describe('from-api-format', () => {
...
+  test('converts empty coClarificationResponses list value to empty string', () => {
+    const nfdivFormat = fromApiFormat({
+      ...results,
+      coClarificationResponses: [],
+    } as unknown as CaseData);
+
+    expect(nfdivFormat).toMatchObject({
+      coClarificationResponses: '',
+    });
+  });
```
Comment (lara-mpt) [inline src/main/app/case/from-api-format.test.ts:327]: is it worth adding in a test for when the list isn't empty? And also below for if a response has been entered?

**Review by lara-mpt — APPROVED** [review]
